src/i18n/en.json

```diff
@@ -964,49 +964,87 @@
+  "actionSidebar.tooltip.streamingPayment.amount": "Funds to be streamed over the chosen \"Amount per\" period.",
+  "actionSidebar.tooltip.streamingPayment.amount.per": "Defines the streaming rate, i.e. per Day, Week, Month. (e.g., 500 CLNY/Month).",
+  "actionSidebar.tooltip.streamingPayment.limit": "The total amount of funds that can be earned for this streaming payment.",
+  "actionSidebar.tooltip.createAgreement.description": "Detailed explanation of the agreement. Including any necessary details, explanations, rationale, and relevant references.",
+  "actionSidebar.tooltip.transferFunds.from": "Team source of the funds.",
+  "actionSidebar.tooltip.transferFunds.to": "Team destination of the funds.",
+  "actionSidebar.tooltip.transferFunds.amount": "Quantity and type of funds.",
+  "actionSidebar.tooltip.mintTokens.amount": "The total amount of new tokens.",
+  "actionSidebar.tooltip.manageTokens.approved": "List of approved tokens in the Colony.",
+  "actionSidebar.tooltip.manageTokens.token": "The name of the tokens visible in the Colony.",
+  "actionSidebar.tooltip.manageTokens.symbol": "The symbol of the corresponding token.",
+  "actionSidebar.tooltip.createNewTeam.team.name": "The name for the new team.",
+  "actionSidebar.tooltip.createNewTeam.team.purpose": "The purpose of the new team.",
+  "actionSidebar.tooltip.createNewTeam.team.colour": "The color used to represent the team.",
+  "actionSidebar.tooltip.editTeam.selectTeam": "The name of the team being changed.",
+  "actionSidebar.tooltip.editTeam.team.name": "The name for the team.",
+  "actionSidebar.tooltip.editTeam.team.purpose": "The purpose of the team.",
+  "actionSidebar.tooltip.editTeam.team.colour": "The color used to represent the team.",
+  "actionSidebar.tooltip.manageReputation.team": "The team reputation is being managed in.",
+  "actionSidebar.tooltip.manageReputation.modification": "The type of modification being made to the members reputation.",
+  "actionSidebar.tooltip.manageReputation.member": "Member or address for the reputation change.",
+  "actionSidebar.tooltip.manageReputation.currentAmount": "Existing amount of reputation.",
+  "actionSidebar.tooltip.manageReputation.newAmount": "New amount of reputation after update.",
+  "actionSidebar.tooltip.manageVerifiedMembers.member": "Member or address being verified.",
+  "actionSidebar.tooltip.managePermissions.member": "Member or address for the permission change.",
+  "actionSidebar.tooltip.managePermissions.team": "The team permissions is being managed in.",
+  "actionSidebar.tooltip.managePermissions.permissions": "The permission for the selected member. View permissions list. INSTRUCTIONS: View permissions list - should open the modal",
```

Comment:
The "INSTRUCTIONS" are what need to be done, not to be included. The text should be:
"The permission for the selected member. View permissions list."

With the text "View permissions list", opening the full permissions modal.

Reply:
done

Reply:
If the "View permissions list." part of the text is not able to be clickable in the popover, can we just remove it please.

Reply:
Ok, removed :)